exercises/beer-song/test/Tests.hs

```diff
@@ -34,7 +34,7 @@ specs = describe "song" $ do
     zipMaybe    []  (y:ys) = (Nothing, Just y ) : zipMaybe [] ys
     zipMaybe (x:xs) (y:ys) = (Just x , Just y ) : zipMaybe xs ys
 
--- Lyrics extracted from `exercism/x-common` on 2016-10-03.
+-- Lyrics extracted from `exercism/problem-specifications` on 2016-10-03.
```

Comment:
a slight lie since problem-specifications did not exist in 2016, but I don't want to change the date and the lie is not serious enough to correct. the point is the student knows where to go to look for the lyrics now, I think?